src/frontend/src/views/Upload/components/Metadata/components/ImportFile/components/Instructions/index.tsx

```diff
@@ -23,14 +24,21 @@ export default function Instructions({ headers, rows }: Props): JSX.Element {
           included in your import file.
         </ListItem>
         <ListItem ordered fontSize="xs">
-          We recommend that you copy your metadata into our{" "}
+          We recommend that you copy your metadata into our
           <StyleDownloadTemplate headers={headers} rows={rows}>
             TSV template
           </StyleDownloadTemplate>
           , but you can use your own TSV or CSV file as well.
         </ListItem>
         <ListItem ordered fontSize="xs">
-          Make sure your column headers match our naming convention.
+          Make sure your column headers match our naming convention or the&nbsp;
```

Comment:
`{" "}` wasn't being respected so i replaced with `&nbsp;` wondering if either @mmmmmmaya or @ehoops-cz knows why and can enlighten me on the reasoning 😆

Reply:
Apparently it's part of the css spec to trim trailing white space when `white-space` is set to normal (the default).  I honestly had no idea there was so much to read about this 😂 .  I usually use `&nbsp` as well.  I think another option may be wrapping the space in a span (iirc) like 
`
<ListItem ordered fontSize="xs">
          Make sure your column headers match our naming convention or the
          <span>{' '}</span>
          <NewTabLink ...
</ListItem>
`

Reply:
interesting! thanks Erin!